src/renderer/js/modal/modalCreditIntro/view.jsx

```diff
@@ -4,55 +4,42 @@ import { CreditAmount, CurrencySymbol } from "component/common";
 import Link from "component/link/index";
 
 const ModalCreditIntro = props => {
-  const { closeModal, totalRewardValue, currentBalance, verifyAccount } = props;
+  const { closeModal, totalRewardValue, currentBalance, addBalance } = props;
   const totalRewardRounded = Math.round(totalRewardValue / 10) * 10;
 
   return (
     <Modal type="custom" isOpen={true} contentLabel="Welcome to LBRY">
       <section>
-        <h3 className="modal__header">{__("Blockchain 101")}</h3>
+        <h3 className="modal__header">
+          {__("Computer Wizard Needs Tokens Badly")}
+        </h3>
         <p>
-          LBRY is controlled and powered by a blockchain asset called{" "}
+          Some actions require{" "}
```

Comment:
this is slightly confusing because it first says: "Some actions require LBC, the blockchain token that powers the LBRY network."

and then it refers to them as credits in the rest of the modal:

"You currently have 0 credits, so the actions you can take are limited.

There are a variety of ways to get credits, including more than 20 credits in free rewards for participating in the LBRY beta."

It makes sense for us because we understand it, but I don't think people who are not familiar with crypto will catch on. I would suggest at least rewording it as follows:

""Some actions require LBRY credits (LBC), the blockchain token that powers the LBRY network."